program/convert-seconds-to-days/convert_seconds_to_days.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+import "fmt"
+
+func main() {
+	var input int
```

Comment:
I suggest you make input as float value because even 86399 input gives you value as 0 but let's get some value as input